Libraries/StyleSheet/NativeColorValueTypesIOS.ios.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+'use strict';
+
+import type {ColorValue} from './ColorValueTypes';
+import {IOSDynamicColorPrivate} from './NativeColorValueTypes';
+
+export type IOSDynamicColorTuple = {
+  light: ColorValue,
+  dark: ColorValue,
+};
+
+export const IOSDynamicColor = (tuple: IOSDynamicColorTuple): ColorValue => {
```

Comment:
(Minor) most components in RN go by stuff like `DatePickerIOS`. Should this match?

Reply:
Hmm, maybe?   I could go either way.   I'd like to hear @TheSavior 's opinion (I think it was Eli who suggested `IOSDynamicColor` originally).